modules/ROOT/pages/cbl-whatsnew.adoc

```diff
@@ -84,9 +121,7 @@ xref:3.1@couchbase-lite:ROOT:cbl-whatsnew.adoc[What's new in previous version 3.
 
 [IMPORTANT]
 --
-There is no 3.2.0 Beta 1 release for the following platforms:
-
-* Java
+There is currently no 3.2.0 Beta 2 release for the following platforms:
```

Comment:
The Vector Search beta release is not available for the following platforms:

Reply:
Done!